screens/EntryScreen.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {Image, StyleSheet, TouchableHighlight, TouchableOpacity, View, Text} from "react-native";
+import baseStyle from "../styles/base";
+import {theme} from "../styles/colors";
+import {AuthActionType, useAuthDispatch} from "../contexts/AuthContext";
+
+export default function EntryScreen(props) {
+    const authDispatch = useAuthDispatch();
+    return (
+        <View style={styles.container}>
+            <Image style={styles.logo} source={require('../assets/seeus-logo.png')}/>
+            <TouchableOpacity style={styles.loginButton} onPress={() => authDispatch({ type: AuthActionType.Login})}>
+                <Text style={styles.loginButtonText}>Login with NetID</Text>
+            </TouchableOpacity>
```

Comment:
NOTE: Need to create a Button component with our styles, so we can reuse it everywhere without manually defining styles I'm currently doing here.